hm_pyhelper/diagnostics/diagnostics_report.py

```diff
@@ -54,29 +53,17 @@ def append_error(self, key):
     def get_errors(self):
         return self[DIAGNOSTICS_ERRORS_KEY]
 
-    def assert_diagnostics_present(self, required_keys: set) -> bool:
+    def get_missing_keys(self, required_keys: set) -> bool:
```

Comment:
return type hint should be changed from bool to list.

Reply:
Good catch, changed: https://github.com/NebraLtd/hm-pyhelper/pull/93/commits/43e1dc7a5230f85053c55c4dc1a9bad460fb55fa

But it should return a set right https://docs.python.org/3/library/stdtypes.html#frozenset.difference